.claude/skills/divio-docs/examples.md

```
@@ -0,0 +1,247 @@
+
+### 2. Start the Development Server
+
+Run the development server:
+
+```bash
+npm run dev
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)